hw/formal/README.md

```diff
@@ -324,6 +324,39 @@ which lists the total number of assertions and the number of proven, vacuous,
 covered and failing assertions for each block. CRASH identifies modules that
 fail to run VC Formal.
 
+## How To Run Connectivity Test on OpenTitan
+
+### Cadence JapserGold
+If you have access to JasperGold's connectivity app from Cadence, there are two
+options to run connectivity test:
+
+#### Batch Script
+You can trigger the `formal_conn.sh` batch script with the following command:
+```
+  cd hw/formal
+  formal_conn.sh -f {path_to_connectivity_csv_file}
+```
+This script also provides the following options:
+* `-top {top_name}`: specify which top_level to run. Default is `top_earlgrey`
+* `-p {core_path}`: specify which fusesoc core file to use. Default is
```

Comment:
Is `-top` needed if you are already providing `-p` switch? Seems like `-p` is a lot more flexible.

Reply:
Thanks for pointing that out Sri. We keep `-top` for top_level names. Then in tcl script, Connectivity test will do: `elaborate -top $env{TOP}`.
I think it should work if the top name is the same as the fusesoc name. But maybe we can keep the option in case the top-level name is different?

Reply:
ok, sounds good